src/backend/egl/display.rs

```diff
@@ -584,7 +588,7 @@ impl EGLDisplay {
         );
         for i in 0..num_planes {
             dma.add_plane(
-                fds[i as usize],
+                unsafe { OwnedFd::from_raw_fd(fds[i as usize]) },
```

Comment:
Might be worth checking the EGL registry for a description of ownership of the returned fds.

Reply:
Yeah. Looks like they are indeed owned by the caller:

https://registry.khronos.org/EGL/extensions/MESA/EGL_MESA_image_dma_buf_export.txt

> 3. Does ownership of the file descriptor remain with the app?
>
> ANSWER: Yes, the app is responsible for closing any fds retrieved.